app/views/summary/UKSitesSummary.scala

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views.summary
+
+import controllers.routes
+import models.{CheckMode, UserAnswers}
+import pages.{AskSecondaryWarehousesPage, PackAtBusinessAddressPage}
+import play.api.i18n.Messages
+import uk.gov.hmrc.govukfrontend.views.Aliases.SummaryList
+import viewmodels.govuk.summarylist._
+import viewmodels.implicits._
+
```

Comment:
This function can be refactored to be more clean by saving your two `SummaryListRowViewModel`  in appropriately named defs outside of your summaryList def. You can then call them in your match cases. 

Pseudo code would be something in the lines of:

```
def summaryList(x,y) = {

    (a,b) match {
         case () =>
             getPackAtBusinessAddressRow
         case () =>
             getAskSecondaryWarehouseRow
         case () => 
             getPackAtBusinessAddressRow
             getAskSecondaryWarehouseRow
    }

}

def getPackAtBusinessAddressRow(...)
def getAskSecondaryWarehouseRow(...)
```